packages/angular/package.json

```diff
@@ -13,15 +13,12 @@
     "serve:dark": "ng serve dev -c dark",
     "storybook": "start-storybook -s .storybook/public -p 6006",
     "storybook:build": "build-storybook -s .storybook/public -c .storybook -o ../../dist/website/storybook/angular --quiet",
-    "postinstall": "ngcc && node ../../scripts/ngcc-clean",
-    "build": "npm-run-all build:lib:cds build:lib:clr build:package:cds build:package:clr build:update:cds build:update:clr",
+    "build": "npm-run-all build:lib:cds build:lib:clr build:update:cds build:update:clr",
     "build:preinstall": "yarn run build:preinstall:cds && yarn run build:preinstall:clr",
-    "build:preinstall:cds": "mkdir -p ../../dist/angular/",
-    "build:preinstall:clr": "mkdir -p ../../dist/clr-angular/",
-    "build:package:clr": "replace '@VERSION' $npm_package_version ./dist/clr-angular/package.json",
-    "build:package:cds": "replace '@VERSION' $npm_package_version ./dist/cds-angular/package.json",
+    "build:preinstall:cds": "mkdir -p ../../dist/angular/ && cp ./projects/cds-angular/package.json ../../dist/angular/",
+    "build:preinstall:clr": "mkdir -p ../../dist/clr-angular/ && cp ./projects/clr-angular/package.json ../../dist/clr-angular/",
```

Comment:
Updated preinstall to copy the package json like the other packages. This ensure yarn installs their dependencies at the top level of the mono repo. Without it yarn updates/installs will fail because the angular package dependencies were never installed.